### PR TITLE
DDCE-5277 Clean up configs

### PR DESCRIPTION
**Pull request by mywyau**
Tried to clean up some of the configuration in the service. Moved configs to FrontendConfig class to all be in one place, so it's easier to know where config values are being used. And mocked out frontendAppConfig in tests.

Idea is to get qa and staging working and cleaned up, then worry about prod. Not sure how to test and check if every config has been resolved.

https://github.com/hmrc/app-config-base/pull/9924 <--- Base
https://github.com/hmrc/app-config-qa/pull/21142 <--- QA
https://github.com/hmrc/app-config-staging/pull/14515 <--- Staging
https://github.com/hmrc/app-config-production/pull/20885 <--- Production

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -107,15 +95,13 @@ session {
 
 mongodb {
   uri                 = "mongodb://localhost:27017/"${appName}
-  timeToLiveInSeconds = 900
```
Comment (lmugnai) [inline conf/application.conf:110]: Why are we removing this?

File: conf/application.conf [hunk 2]
```diff
@@ -150,10 +132,9 @@ tracking-consent-frontend {
...
   callbackBaseUrl = "http://localhost:12600"
   minFileSize = "1b"
-  maxFileSize = "5000000b"
   maxFiles    = 5
+  maxFileSize = "5000000b"
 }
 
 internal-auth.token = "82fac546-6de3-47bb-a1d7-357d68ecd3ac"
 object-store.default-retention-period = 1-month
-create-internal-auth-token-on-start = true
```
Comment (lmugnai) [inline conf/application.conf:159]: Why are we removing this?

File: conf/application.conf (hunk 1 above)
Comment (mywyau) [reply conf/application.conf:110]: don't think this is being used. the only frontend mongo repository is counter and has no ttl from what can tell. Unless this is being used in some other way perhaps?

File: conf/application.conf (hunk 2 above)
Comment (mywyau) [reply conf/application.conf:159]: I'll add this back in as it's used and got accidentally removed.

File: conf/application.conf (hunk 1 above)
Comment (mywyau) [reply conf/application.conf]: As discussed adding back in due to feeling uneasy when it comes to mongo and ticket is to just fix the warnings. Small clean up for now but leaving it in.

**Review by lmugnai — APPROVED** [review]

**Pull request merged**